bin/console

```diff
@@ -28,7 +32,11 @@ if ($input->hasParameterOption('--no-debug', true)) {
     putenv('APP_DEBUG='.$_SERVER['APP_DEBUG'] = $_ENV['APP_DEBUG'] = '0');
 }
 
-(new Dotenv())->bootEnv(dirname(__DIR__).'/.env');
+if (getenv('ENV_DIR') !== false) {
+    (new Dotenv())->bootEnv(getenv('ENV_DIR').'/.env');
+} else {
+    (new Dotenv())->bootEnv(dirname(__DIR__).'/.env');
+}
```

Comment:
How about something like

```php
$env_dir = getenv('ENV_DIR') ?? dirname(__DIR__);
```

at the start of the file, and use the variable below (for the $_SERVER + the Doenv() call)?

Reply:
That sounds good, should we also add the `!= ''`?

Reply:
`!=` does the job here (note the difference between `!==` which checks the type with `!=` which does not, and an empty string is considered "falsy")